app/jobs/bulkrax/importer_job.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+# OVERRIDE Bulkrax 1.0.2 to rescue errors
+
+require_dependency Bulkrax::Engine.root.join('app', 'jobs', 'bulkrax', 'importer_job').to_s
+
+  Bulkrax::ImporterJob.class_eval do
+    def perform(importer_id, only_updates_since_last_import = false)
+      importer = Bulkrax::Importer.find(importer_id)
+
+      importer.current_run
+      unzip_imported_file(importer.parser)
+      import(importer, only_updates_since_last_import)
+      update_current_run_counters(importer)
+      schedule(importer) if importer.schedulable?
+      Bulkrax::IndexAfterJob.set(wait: 1.minute).perform_later(importer)
```

Comment:
I agree about the comment. I'll add it.

we'd still need the retry logic because its not just after this job, but after all the import jobs for a given import set.

Reply:
👍